lms/djangoapps/certificates/views/webview.py

```diff
@@ -261,6 +261,8 @@ def _update_course_context(request, context, course, course_key, platform_name):
             partner_short_name=context['organization_short_name'],
             platform_name=platform_name)
 
+    context['course_description'] = get_course_about_section(request, course, 'short_description')
```

Comment:
A test case here would be great to ensure we don't remove this feature by mistake.

Reply:
Done